INSTALL.md

```diff
@@ -49,6 +49,42 @@ make test
 If you want to get rid of this environment, just run `make clean`.
 
 
-#### Development Environment Setup 
+## Development Environment Setup on Mac OS X
 
-//TODO: Coming soon...
\ No newline at end of file
+### Dependencies
+
+First you should install following things:
+
+XCode: Install it via the Apple Store application or in [Apple's developer site](https://developer.apple.com/xcode/download/)
```

Comment:
Version 9.4 to be precise, it will fail with v10.

Reply:
i just came here to write that ;) 

https://github.com/thoughtbot/capybara-webkit/issues/1071